torch/_inductor/lowering.py

```diff
@@ -1404,9 +1404,6 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten.zeros.names)
 
 
-add_layout_constraint(aten.convolution, constrain_to_fx_strides)
```

Comment:
this was necessary for convnext (even when convnext is not run with channels-last option), can you check that it's perf is not regressed? I think it's now done anyway in `require_stride_order`

Reply:
Will do.